parlemonde/1village
# feat: add new cultural indicator activity

### Motivation

To solve this ticket : https://trello.com/c/uqMzV4oc/26-cr%C3%A9er-un-indice-culturel

### Changes

- Add new pages to create activity
- Add new activity type
- Add new Icon for this activity
- Update navigation links
- Modify activity card to display cultural indicator

### Test

- `git checkout feat/cultural-index`
- In navigation component you should see "Présenter un indice culturel"
- Click on link, you should be redirect to create cultural indicator
- Fill the form and try to play with this activity
- Fill the form until step 3 whithout publish, you should see that draft version is save (go to "mes activités" to verify)
- Publish and try to answer 
- Check the behavior of the new activity and its card.
